.github/workflows/ci.yml

```diff
@@ -1,7 +1,7 @@
 on:
   push:
     branches:
-      - master
+      - v0.3.5
```

Comment:
To be removed, just forcing CI to be run, since this PR is not to be merged in master